lib/codegen/src/timing.rs

```diff
@@ -62,6 +62,7 @@ define_passes!{
     gvn: "Global value numbering",
     licm: "Loop invariant code motion",
     unreachable_code: "Remove unreachable blocks",
+    // constant_folding: "Fold constant expressions",
```

Comment:
Uncommenting this to add `constant_folding` pass timing will not compile. The array generated by the macro is currently 32 items in length, and a static slice of 33 items does not implement `Default`. When const generics land, this will be fixed, but not sure what to do about it before then.

Reply:
Huh. Not sure what to do yet either.

Reply:
Figured it out. Just had to coerce them to an dynamically sized slice.